### ISSUE TITLE
Passive event listeners

### ISSUE THREAD

**Issue by johnhult** [google-map-react/google-map-react#625]
Not sure if it's this library but according to [https://issuetracker.google.com/issues/63211698](url) Google Map should not give warnings when doing a Lighthouse Audit for not using passive eventListeners. Is this something in regards to this library? Doing a lighthouse audit I get the following error:

<img width="694" alt="screen shot 2018-08-07 at 14 22 59" src="https://user-images.githubusercontent.com/1528648/43775549-7292342c-9a4d-11e8-943d-f69463acf05e.png">

If it indeed isn't this library then feel free to close this issue.

**Comment by itsmichaeldiego** [comment]
@johnhult Thanks for your help. It seems that its the google map api itself that its throwing an error, you can see the URL of the warning is actually the url of googleapis and not your URL, if it would be our library it will be showing the URL of your current project.

**Comment by johnhult** [comment]
Yeah I thought it was kinda weird. What version of google maps is this library using? As mentioned in my comment version 3.32 of google maps should apparently not give these errors. But this uses a higher version than that, right?

I'll try and look into it a bit more aswell but just want to make sure I have the correct assumptions first 😄

**Comment by itsmichaeldiego** [comment]
@johnhult Our library uses latest released version always, you can specify the version in `bootstrapURLKeys` doing like: 

```
<GoogleMapReact
  ...
  bootstrapURLKeys={{
    ...
    v: '3.33',
  }}
/>
```

**Comment by itsmichaeldiego** [comment]
@johnhult @jooj123 Fix #631 has been merged and released in `1.0.7`

Let me know if it works.
